analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -152,11 +152,13 @@ class GoMod(
 
         fun parseModuleEntry(entry: String): Identifier =
             entry.substringBefore('@').let { moduleName ->
+                val version = moduleInfo(moduleName).version
+
                 Identifier(
-                    type = managerName,
+                    type = managerName.takeIf { version.isBlank() } ?: "Go",
```

Comment:
I don't completely get this. Why is `GoMod` used as `type` if the version is blank?
Why not use `"Go"` all the time?

Reply:
There is just one project, all other entries are packages. Only the project has an empty version string. For the project GoMod shall be used.